app/AbstractKernel.php

```diff
@@ -99,7 +97,16 @@ public function registerContainerConfiguration(LoaderInterface $loader)
         $loader->load(
             $this->rootDir . DIRECTORY_SEPARATOR
             . 'config' . DIRECTORY_SEPARATOR
-            . $this->getContext() . DIRECTORY_SEPARATOR
+            . 'config.yml'
+        );
+        $loader->load(
+            $this->rootDir . DIRECTORY_SEPARATOR
+            . 'config' . DIRECTORY_SEPARATOR
+            . 'config_' . $this->getContext() . '.yml'
+        );
+        $loader->load(
+            $this->rootDir . DIRECTORY_SEPARATOR
+            . 'config' . DIRECTORY_SEPARATOR
```

Comment:
@webmozart has proposed this structure to eliminate differences between symfony-standard and sulu.